tests/test_cli.py

```diff
@@ -842,9 +844,9 @@ def test_recover_recovery_staging_path(
         mock_recover_args,
         backup_is_incremental,
         backup_is_compressed,
-        recovery_staging_path_arg,
```

Comment:
I think we should have different tests for dealing with `recovery_staging_path` and with `local_staging_path`.